applications/server/views/log_test/log_report.html

```diff
@@ -0,0 +1,19 @@
+{{extend 'layout.html'}}
+<script src="/server/static/js/dygraph-combined-dev.js"></script>
+<h1>Visualization Test</h1>
+<center>
+    <div id="graphdiv" style="width: 600px; height: 400px"></div>
+</center>
+<script type="text/javascript">
+    g = new Dygraph(
+        document.getElementById("graphdiv"),
+        "{{=URL('static', 'temper.csv')}}",
```

Comment:
Check in here means adding the file into git. Your branch (visual_team) doesn't contain this file in the git repo.
Did you perform git add temper.csv after creating this file?